src/lib/dom-api.html

```diff
@@ -752,7 +750,35 @@
         return n$ ? Array.prototype.slice.call(n$) : [];
       };
 
-      DomApi.prototype._distributeParent = function() {}
+      DomApi.prototype._distributeParent = function() {};
+
+      DomApi.prototype.notifyObservers = function() {
+        if (this.node.shadowRoot) {

```

Comment:
Need help understanding
* Why notify only shadow root observers?
* Where is notifyObservers for shady?
* Notify only schedules notify... not a flush?